### DISCUSSION TITLE
Table export option

### DISCUSSION THREAD

**Discussion by Rmfjd1978** [h2oai/wave#1702] [Q&A]
When i'm using the download do CSV option from a table the enconding is not correct how can i solve this?

screen capture:
![image](https://user-images.githubusercontent.com/98091225/202173723-7b9e64d2-76a4-47aa-860d-1d265ab7a521.png)

csv headers:
Objecto,"Estado","Data Envio","Data ReceÃ§Ã£o","DestinatÃ¡rio","Remetente"

**Comment by marek-mihok** [comment]
Thank you @Rmfjd1978 for letting us know!

The problem is, that unicode is not supported for table export in the latest version of Wave (wave-0.23.1).

The good news is [this issue is already fixed](https://github.com/h2oai/wave/pull/1674) and the new release is just around the corner (it should be the matter of days).

If you need the correct exporting now, feel free to try our [nightly builds](https://wave.h2o.ai/docs/installation#early-adopters).
